src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Responses/GetStandardsListItem.cs

```diff
@@ -11,14 +11,14 @@ public class GetStandardsListItem
         public string Title { get; set; }
         public int Level { get; set; }
         public decimal Version { get; set; }
-        [JsonIgnore] 
-        public long MaxFunding => GetMaxFunding();
+        [JsonIgnore]
+        public long MaxFunding => GetFundingDetails("MaxFunding");
```

Comment:
```suggestion
        public long MaxFunding => GetFundingDetails(nameof(MaxFunding));
```

Not quite sure why this is a long? would make sense for it to be an int as thats whats coming from the inner API